src/commands/misc/uigradient.js

```diff
@@ -0,0 +1,41 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+const { Attachment } = require('discord.js')
+const snekfetch = require('snekfetch')
+const { createCanvas } = require('canvas')
+
+module.exports = class UIGradient extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'uigradient'
+    this.aliases = ['rg', 'randomgradient']
+    this.category = 'general'
+  }
+
+  async run ({ t, author, channel }) {
+    const embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+
+    const { body } = await snekfetch.get('https://cdn.jsdelivr.net/gh/ghosh/uiGradients/gradients.json')
+    const { name, colors } = body[Math.floor(Math.random() * body.length)]
```

Comment:
put it on canvastemplates

Reply:
This could be done as a general Util - who knows when we'll need gradients somewhere? The function could take the colors + the width and height for the gradient.